lib/tether/toolbox.go

```diff
@@ -237,3 +270,161 @@ func (t *Toolbox) halt() error {
 
 	return session.Cmd.Process.Kill()
 }
+
+// ----------
```

Comment:
Function comments don't need to have extra formatting - please remove lines 274, 76 and 77.